src/include/pbs_db.h

```diff
@@ -866,6 +531,21 @@ int pg_db_delete_svrattr(pbs_db_conn_t *conn, pbs_db_obj_info_t *obj);
  */
 int compare_obj_hash(void *qs, int len, void *oldhash);
 
+/**
+ * @brief
+ *	Function to create new databse user or change password of current user.
+ *
+ * @param[in] conn[in]: The database connection handle which was created by pbs_db_connection.
+ * @param[in] user_name[in]: Databse user name.
+ * @param[in] password[in]:  New password for the database.
+ * @param[in] olduser[in]: old database user name.
+ *
+ * @retval       -1 - Failure
+ * @retval        0  - Success
+ *
+ */
+int pbs_db_password(void *conn, char *userid, char *password, char *olduser);
+
 /**
  * @brief
  *	Frees attribute list memory
```

Comment:
free_db_attr_list() is also NOT a DB API that libdb will implement. In fact it is implemented in pbs_db_func.c, so put it in a different header file